requirements.txt

```diff
@@ -1 +1,2 @@
-pygerduty==0.25
+pypd==0.1.0
+urllib3[secure]
```

Comment:
Why is `urllib3` here? Doesn't look like it's used by the pack anywhere? Leftover from something else?